ledger/ledger-api-tests/suites/src/main/scala/com/daml/ledger/api/testtool/suites/v1_dev/ExplicitDisclosureIT.scala

```diff
@@ -255,17 +255,10 @@ final class ExplicitDisclosureIT extends LedgerTestSuite {
       _ <- ledger.exercise(owner, testContext.delegatedCid.exerciseArchive())
 
       // Exercise the choice using the now inactive disclosed contract
-      exerciseError <- testContext
+      _ <- testContext
         .exerciseFetchDelegated(testContext.disclosedContract)
         .mustFail("the contract is already archived")
-    } yield {
-      assertGrpcError(
-        exerciseError,
-        LedgerApiErrors.ConsistencyErrors.ContractNotFound,
-        None,
-        checkDefiniteAnswerMetadata = true,
-      )
-    }
```

Comment:
Error codes are diverging (e.g. Canton is returning `LOCAL_VERDICT_INACTIVE_CONTRACTS`). Do not assert the specific error code for PoC s sake